socs/agents/hwp_supervisor/agent.py

```diff
@@ -1225,6 +1312,64 @@ def abort_action(self, session, params):
         session.data['action'] = action.encode()
         return True, "Set state to idle"
 
+    def enable_driver_board(self, session, params):
+        """enable_driver_board()
+
+        **Task** - Enables the HWP driver board
+
+        Notes
+        --------
+
+        Example of ``session.data``::
+
+            >>> session['data']
+            {'action':
+                {'action_id': 3,
+                'completed': True,
+                'cur_state': {'class': 'Done', 'msg': None, 'success': True},
+                'state_history': List[ConrolState],
```

Comment:
Does this `state_history` include timestamps? That's one thing to include in `session['data']` generally, else it's difficult to tell if the data is fresh.

Reply:
In the most recent commit I added a "last_update_time" to the base state, which should tell you how fresh the state is. Hope that's sufficient.